gst/mqtt/mqttsink.c

```diff
@@ -42,8 +57,21 @@ enum
   DEFAULT_NUM_BUFFERS = -1,
   DEFAULT_QOS = TRUE,
   DEFAULT_SYNC = FALSE,
+  DEFAULT_MQTT_OPT_CLEANSESSION = TRUE,
+  DEFAULT_MQTT_OPT_KEEP_ALIVE_INTERVAL = 60,    /* 1 minute */
+  DEFAULT_MQTT_DISCONNECT_TIMEOUT = 3000,       /* 3 secs */
+  DEFAULT_MQTT_PUB_WAIT_TIMEOUT = 1,    /* 1 secs */
```

Comment:
maybe better to use the same timeout unit?

Reply:
@dongju-chae That would be better. Thanks. 

However, it should be addressed in another PR to avoid confusion from the changes in the property. After the demonstration, we will refactor this whole code.

Reply:
Yeap, I agree.